[DEV] 'make test' is broken on develop

When using poetry 0.12.x to work with the current `develop` branch, the `make test` is broken, i.e.

```
$ poetry --version
Poetry 0.12.17
$ make format 
All done! ✨ 🍰 ✨
331 files left unchanged.
$ make test
make: py.test: Command not found
Makefile:35: recipe for target 'test' failed
make: *** [test] Error 127
```

git log at the time of this issue:
```
$ git ll
* 7b13f00 - (HEAD -> develop, upstream/develop) Optimize tests/conftest.py (#1271) (7 hours ago) <Darren Weber>
* 7b1e01f - Add make targets for clean and format (#1266) (7 hours ago) <Darren Weber>
* 9b51606 - Fix file installation for wheels with no Requires-Dist (#1322) (2 days ago) <Arun Babu Neelicattu>
* c5e54ad - Fix support for url sourced packages in pip installer (#1306) (2 days ago) <Arun Babu Neelicattu>
* 8c0e1d6 - Fix tests (3 days ago) <Sébastien Eustace>
*   27dbf0b - Merge branch 'master' into develop (3 days ago) <Sébastien Eustace>
|\  
| * 0b38fad - Setup CI with Github Actions (#1347) (3 days ago) <Sébastien Eustace>
| * cd3ad8c - Update README for pipenv command name (#1343) (4 days ago) <Grey Li>
| * 8db8454 - Improve documentation (#1344) (4 days ago) <Grey Li>
| *   f4803e9 - Merge pull request #1309 from sdispater/950-simplified (2 weeks ago) <Bryce Drennan>
```

--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,7 @@ setup: setup-python
 
 # test your application (tests in the tests/ directory)
 test:
-	@py.test --cov=poetry --cov-config .coveragerc tests/ -sq
+	@poetry run pytest --cov=poetry --cov-config .coveragerc tests/ -sq
 
 release: build linux_release osx_release
 

